test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -15,16 +15,19 @@
     public void elementsAreUnique() throws Exception {
         // empty list
         assertAreUnique();
+        assertIsAnyNull();
```

Comment:
These assertions should be done in a separate test and not within the elementsAreUnique test.